src/kudu/security/security_flags.cc

```diff
@@ -36,7 +36,7 @@ const char* const SecurityDefaults::SecurityDefaults::kDefaultTlsCiphers =
                                    "AES256-SHA256:AES128-SHA256:"
                                    "AES256-SHA:AES128-SHA";
 
-const char* const SecurityDefaults::SecurityDefaults::kDefaultTlsMinVersion = "TLSv1";
+const char* const SecurityDefaults::SecurityDefaults::kDefaultTlsMinVersion = "TLSv1.2";
```

Comment:
From Production... TLSV1.2 is what is negotiated, so this is a safe change. Maybe we can add that to the test plan.

0320 18:58:24.803942 (+    91us) client_negotiation.cc:511] Negotiated TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 TLSv1.2 Kx=ECDH Au=RSA Enc=AESGCM(256) Mac=AEAD

Reply:
Yes, the minimum TLS version is 1.2 in all Meta products.